### ISSUE TITLE
Invalid client id 

### ISSUE THREAD

**Issue by hazy-kun** [crunchy-labs/crunchy-cli#382]
**Describe the bug**
Can't download anymore in archive mode even in download mode cause Invalid client id

**To Reproduce**
Steps / command to reproduce the behavior:
```
:: An error occurred: Invalid client id (https://cr-play-service.prd.crunchyrollsvc.com/v1/G9DUE5Q41/console/switch/play)
Errore nel download del file: Command 'C:\crunchyrust\crunchy.exe archive --include-fonts --include-chapters --merge audio -s en-US -o "[CR] {series_name} S{season_number}E{episode_number} {title}.mkv" https://www.crunchyroll.com/series/GKEH2G0P9/stardust-telepath' returned non-zero exit status 1.
```

**Client (please complete the following information):**
 - OS: Windows
 - Version [v3.4.3]

**Comment by nero-0** [comment]
Same here.

**Comment by jamel-hrt** [comment]
Same here. I tried the v3.4.3 and v3.4.2 version

**Comment by hazy-kun** [comment]
Maybe DRM-FREE endpoint is dead

**Comment by RezoDwB** [comment]
I can also confirm with the current version.

Running in PowerShell, error occurs in both archive and download.

PS C:\Users\brent> crunchy-cli archive -a ja-JP -a en-US -o "{series_name} [S{season_number}E{episode_number}].mkv" https://www.crunchyroll.com/series/G69PZ5PDY/overlord
:: — Logging in                                                                                                                                                                                                                                    :: \ Logging in                                                                                                                                                                                                                                    :: | Logging in                                                                                                                                                                                                                                    :: / Logging in                                                                                                                                                                                                                                    :: — Logging in                                                                                                                                                                                                                                    :: \ Logging in                                                                                                                                                                                                                                    :: | Logging in                                                                                                                                                                                                                                    :: √ Logging in                                                                                                                                                                                                                                    :: — Parsing url 1                                                                                                                                                                                                                                 :: \ Parsing url 1                                                                                                                                                                                                                                 :: | Parsing url 1                                                                                                                                                                                                                                 :: / Parsing url 1                                                                                                                                                                                                                                 :: — Parsing url 1                                                                                                                                                                                                                                 :: \ Parsing url 1                                                                                                                                                                                                                                 :: | Parsing url 1                                                                                                                                                                                                                                 :: / Parsing url 1                                                                                                                                                                                                                                 :: — Parsing url 1                                                                                                                                                                                                                                 :: \ Parsing url 1                                                                                                                                                                                                                                 :: | Parsing url 1                                                                                                                                                                                                                                 :: √ Parsed url 1                                                                                                                                                                                                                                  :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: → Series Overlord is not available with ar-ME, ar-SA, ca-ES, de-DE, en-IN, es-ES, es-LA, fr-FR, hi-IN, id-ID, it-IT, ja-JP, ko-KR, ms-MY, pl-PL, pt-PT, ta-IN, te-IN, th-TH, tr-TR, vi-VN, zh-CN, zh-CN, zh-TW subtitles
:: \ Fetching series details                                                                                                                                                                                                                       :: | Fetching series details                                                                                                                                                                                                                       :: / Fetching series details                                                                                                                                                                                                                       :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: | Fetching series details                                                                                                                                                                                                                       :: / Fetching series details                                                                                                                                                                                                                       :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: | Fetching series details                                                                                                                                                                                                                       :: / Fetching series details                                                                                                                                                                                                                       :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: | Fetching series details                                                                                                                                                                                                                       :: / Fetching series details                                                                                                                                                                                                                       :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: √ Loaded series information for url 1                                                                                                                                                                                                           :: Overlord Season 2 (Overlord II)
::      1. The dawn of Despair » S02E01
::      2. Departure » S02E02
::      3. Lizard men, gathering » S02E03
::      4. Army of Death » S02E04
::      5. The freezing god » S02E05
::      6. Those who pick up, those who are picked up » S02E06
::      7. Blue roses » S02E07
::      8. A boy's feeling » S02E08
::      9. Soaring sparks of fire » S02E09
::      10. Disturbance begins in the royal capital » S02E10
::      11. Jaldabaoth » S02E11
::      12. The final Battle of the disturbance » S02E12
::      13. The ultimate trump card » S02E13
:: Overlord Season 3 (Overlord III)
::      1. A Ruler's Melancholy » S03E01
::      2. Carne Village Once More » S03E02
::      3. Enri’s Upheaval and Hectic Days » S03E03
::      4. Giant of the East, Demon Snake of the West » S03E04
::      5. Two Leaders » S03E05
::      6. Invitation to Death » S03E06
::      7. Butterfly Entangled in a Spider’s Web » S03E07
::      8. A Handful of Hope » S03E08
::      9. War of Words » S03E09
::      10. Preparation for War » S03E10
::      11. Another Battle » S03E11
::      12. Massacre » S03E12
::      13. Player vs Player » S03E13
:: Overlord Season 4 (Overlord IV)
::      1. Sorcerer Kingdom Ains Ooal Gown: Ains Ooal Gown Nation of Leading Darkness » S04E01
::      2. Re-Estize Kingdom » S04E02
::      3. Baharuth Empire » S04E03
::      4. The Ruler of Conspiracy » S04E04
::      5. In Pursuit of the Land of Dwarves » S04E05
::      6. The Impending Crisis » S04E06
::      7. Frost Dragon Lord » S04E07
::      8. An Unexpected Move » S04E08
::      9. Countdown to Extinction » S04E09
::      10. The last king » S04E10
::      11. Well-prepared Traps » S04E11
::      12. Invasion of the royal capital » S04E12
::      13. The Witch of the Falling Kingdom » S04E13
:: An error occurred: Invalid client id (https://cr-play-service.prd.crunchyrollsvc.com/v1/G6K503J9Y/console/switch/play)
PS C:\Users\brent> crunchy-cli archive -a ja-JP -a en-US -o "{series_name} [S{season_number}E{episode_number}].mkv" https://www.crunchyroll.com/watch/GN7UDGW07/the-dawn-of-despair
:: — Logging in                                                                                                                                                                                                                                    :: \ Logging in                                                                                                                                                                                                                                    :: | Logging in                                                                                                                                                                                                                                    :: / Logging in                                                                                                                                                                                                                                    :: — Logging in                                                                                                                                                                                                                                    :: \ Logging in                                                                                                                                                                                                                                    :: | Logging in                                                                                                                                                                                                                                    :: √ Logging in                                                                                                                                                                                                                                    :: — Parsing url 1                                                                                                                                                                                                                                 :: \ Parsing url 1                                                                                                                                                                                                                                 :: | Parsing url 1                                                                                                                                                                                                                                 :: / Parsing url 1                                                                                                                                                                                                                                 :: — Parsing url 1                                                                                                                                                                                                                                 :: \ Parsing url 1                                                                                                                                                                                                                                 :: √ Parsed url 1                                                                                                                                                                                                                                  :: — Fetching series details                                                                                                                                                                                                                       :: \ Fetching series details                                                                                                                                                                                                                       :: | Fetching series details                                                                                                                                                                                                                       :: → Episode 1 is not available with ar-ME, ar-SA, ca-ES, de-DE, en-IN, es-ES, es-LA, fr-FR, hi-IN, id-ID, it-IT, ja-JP, ko-KR, ms-MY, pl-PL, pt-PT, ta-IN, te-IN, th-TH, tr-TR, vi-VN, zh-CN, zh-CN, zh-TW subtitles
:: | Fetching series details                                                                                                                                                                                                                       :: √ Loaded series information for url 1                                                                                                                                                                                                           :: Overlord Season 2 (Overlord II)
::      1. The dawn of Despair » S02E01
:: An error occurred: Invalid client id (https://cr-play-service.prd.crunchyrollsvc.com/v1/G6K503J9Y/console/switch/play)

**Comment by zinezockt** [comment]
Same here

**Comment by bytedream** [comment]
This error differs from the DRM problems we already had. Luckily I have a unpatched switch that is able to run CFW, I'll try if I can intercept the switchs' traffic and see what the requirements for a correct client id are.

**Comment by gleidsonnunes** [comment]
Same here

**Comment by bytedream** [comment]
The issue got fixed in [v3.5.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.5.0).
If you've used a package manager to install crunchy-cli, it might take some time until the update is available for it.

**Comment by NSExceptional** [comment]
@bytedream @frootaside we could use some help over at https://github.com/yt-dlp/yt-dlp/issues/9745 with a similar issue if either of y'all have the bandwidth 🙏🏻
